src/components/CompareResults/ResultsTable.tsx

```diff
@@ -36,12 +36,25 @@ const cellsConfiguration: CompareResultsTableConfig[] = [
   },
   {
     key: 'comparisonSign',
-
     gridWidth: '0.2fr',
   },
   {
     name: 'New',
     key: 'new',
+    gridWidth: '1fr',
+  },
+  {
+    name: 'Base`σ`',
+    key: 'basestandarddeviation',
+    gridWidth: '1fr',
+  },
+  {
+    key: 'comparisonSign',
+    gridWidth: '0.2fr',
+  },
+  {
+    name: 'New`σ`',
+    key: 'newstandarddeviation',
```

Comment:
Look great! 

However, could you please add the full phrase "standard deviation" vs the symbol?

Reply:
sure I can do that as well But the text was taking more space and the UI was looking stuffy @Carla-Moz

Reply:
The new changes are looking like this @Carla-Moz 
![image](https://github.com/user-attachments/assets/49de5f40-2a91-43ea-aed4-201b5b3df1ba)